src/index.js

```diff
@@ -23,7 +23,16 @@ Chart._adapters._date.override({
 	_create: function(time) {
 		return DateTime.fromMillis(time, this.options);
 	},
-
+	
+	/**
+	 * @private
+	 */
+	_isoWeekday: function(dateTime, weekday){
```

Comment:
there should be a space before the `{` for consistency with the existing code style

Reply:
done